### PR TITLE
[5.x] Encrypt content attribute at rest

### PR DESCRIPTION
**Pull request by gawsoftpl**
Pull request for #1532 [Feature] Encryption at the Rest

### CODE REVIEW

**Comment by crynobone** [comment]
Marking this as draft since tests are failing

File: src/Storage/EntryModelEncryptionCast.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    }
+
+    /**
+     * Check that encryption is enabled.
+     *
+     * @return bool
+     */
+    protected function encryptionIsEnabled()
+    {
+        return config('telescope.encryption');
```
Comment (crynobone) [inline src/Storage/EntryModelEncryptionCast.php]: On existing application with published config this would return `null`.

**Review by crynobone — CHANGES_REQUESTED** [review]

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If applicable, please consider releasing your code as a package so that the community can still take advantage of your contributions!

**Comment by gawsoftpl** [comment]
I can create new package. Best way will be use ModelObserver but I dont know why ModelOberser not working with [EntryModel.php](https://github.com/laravel/telescope/blob/5.x/src/Storage/EntryModel.php)

Not fired `creating` hook, retrieved working fine.

**Comment by gawsoftpl** [comment]
> Thanks for your pull request to Laravel!
> 
> Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include.
> 
> If applicable, please consider releasing your code as a package so that the community can still take advantage of your contributions!

I can create new package. Best way will be use ModelObserver but I dont know why ModelOberser not working with [EntryModel.php](https://github.com/laravel/telescope/blob/5.x/src/Storage/EntryModel.php)

Not fired `creating` hook, retrieved working fine.

**Pull request closed without merging**